src/libcollections/vec.rs

```diff
@@ -1165,6 +1165,15 @@ impl<T> Vec<T> {
         }
         other
     }
+    
+    pub fn remove_item<T: PartialEq>(&mut self, item: T) {
```

Comment:
I'm sorry, I'm having some issues with github at the moment, Im working on it, I'm an absolute newbie to github and I'm trying to fix things